tests/memory_maps/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+define_shared_lib(
+  LIBNAME 2
+  SRCS 2.c
+  PKEY 2
+)
+
+define_shared_lib(
+  LIBNAME 3
+  SRCS 3.c
+  PKEY 3
+)
+
+define_shared_lib(
+  LIBNAME 4
+  SRCS 4.c
+  PKEY 4
+)
+
+define_test(
+  SRCS main.c
+  PKEY 1
+  LIBS 2 3 4
+  NEEDS_LD_WRAP
+  CRITERION_TEST
+  WITHOUT_SANDBOX
```

Comment:
Is memory-map debugging incompatible with the tracer? Document here whether that's fundamental or blocked on fixing a particular bug.

Reply:
Hmm, I'm not sure, but the `fopen("/proc/self/maps", "r")` is failing.

Reply:
Ok, tracked at #594.

Reply:
Is this good to merge then?  And we'll fix the tracer separately?

Reply:
Yep, go for it.